securedrop/models.py

```diff
@@ -35,6 +37,14 @@
 ARGON2_PARAMS = dict(memory_cost=2**16, rounds=4, parallelism=2)
 
 
+def generate_otp_secret() -> str:
+    """
+    Generate an OTP secret of 160 bits encoded base32
+    """
+    symbols = string.ascii_uppercase + string.digits
+    return ''.join(secrets.choice(symbols) for i in range(32))
```

Comment:
For readability it might make sense to define the secret length like `OTP_LENGTH = 32` and use that throughout.

Reply:
i was in favour of the simple number and a comment to not confuse further developers thinking that this could be tweaked.